test/intl402/DateTimeFormat/prototype/formatToParts/fractionalSecondDigits.js

```diff
@@ -31,15 +32,6 @@ function assertParts(parts, minute, second, fractionalSecond, message) {
   }
 }
 
-assert.throws(RangeError, () => {
-  new Intl.DateTimeFormat(
-    'en', { minute: "numeric", second: "numeric", fractionalSecondDigits: 0});
-}, "fractionalSecondDigits 0 should throw RangeError for out of range");
-
-assert.throws(RangeError, () => {
-  new Intl.DateTimeFormat(
-    'en', { minute: "numeric", second: "numeric", fractionalSecondDigits: 4});
-}, "fractionalSecondDigits 4 should throw RangeError for out of range");
 
 let dtf = new Intl.DateTimeFormat(
```

Comment:
Should this one test the same values that you're looping over in the other test files? (undefined, 0, "0", etc)